src/main/java/ru/ayaz/MessageProcessor.java

```diff
@@ -0,0 +1,28 @@
+package ru.ayaz;
+
+public class MessageProcessor {
+
+    public boolean isCommand(String message) {
+        message = message.trim();
+        return message.startsWith("#");
+    }
+
+    public String getCommand(String message) {
+        message = message.trim();
+        if (message.contains(" ")) {
+            return message.substring(0, message.indexOf(" "));
+        } else {
+            return message;
+        }
+    }
+
+    public String getFirstArgument(String message) {
+        message = message.trim();
```

Comment:
Same here, create a local variable. You might want to use your function parameter elsewhere in the function to read the value from it, but it will be changed and you won't even notice it.